lib/modules/apostrophe-assets/index.js

```diff
@@ -575,6 +575,17 @@ module.exports = {
         var last = chain[chain.length - 1];
         var from = self.getAssetRoot() + '/public/modules/' + name;
         var to = last.dirname + '/public';
+        var scope = to.match(/@.+?(?=\/)/);
+
+        // Create symlink folder for scoped package, if it does not exist.
+        if (scope) {
+          var symlink = self.getAssetRoot() + '/public/modules/' + scope[0];
```

Comment:
I don't like to repeat myself. But it could be faster to not set this variable. Instead repeat this path like on lines `558` and `559`?